src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/FileContentProcessor.java

```diff
@@ -96,6 +97,14 @@ public void processZipFileContent(
         } catch (PreviouslyFailedToUploadException ex) {
             log.warn("Rejected file {} from container {} - failed previously", zipFilename, containerName, ex);
             createEvent(DOC_UPLOAD_FAILURE, containerName, zipFilename, ex.getMessage());
+        } catch (OcrValidationServerSideException ex) {
+            log.error("Failed to process file {} from container {}, "
+                          + "message from OCR validation endpoint {} - will retry",
+                      zipFilename,
+                      containerName,
+                      ex.getMessage(),
```

Comment:
I don't think logging ex.getMessage is required, it will be anyway logged as the whole ex is passed.